numpy/core/numeric.py

```diff
@@ -655,8 +656,6 @@ def require(a, dtype=None, requirements=None):
     else:
         subok = True
 
-    arr = array(a, dtype=dtype, copy=False, subok=subok)
-
     copychar = 'A'
     if 'FORTRAN' in requirements or \
```

Comment:
The line continuation should be removed and replaced by a parenthesized expression. I'd also be tempted to just take the first character above when the options are uppercased as the single character versions are documented as valid.